python/pyhail/tests.py

```diff
@@ -208,6 +208,43 @@ def test_dataset(self):
         sample2_split.variant_qc().print_schema()
         
         sample2.variants_to_pandas()
-        
+
+        sample_split.annotate_variants_expr("va.nHet = gs.filter(g => g.isHet).count()")
+
+        kt = sample_split.aggregate_by_key("Variant = v", "nHet = g.map(g => g.isHet.toInt).sum().toLong")
+
+    def test_keytable(self):
+        # Import
+        kt = self.hc.import_keytable(self.test_resources + '/sampleAnnotations.tsv', 'Sample', config = TextTableConfig(impute = True))
+        kt2 = self.hc.import_keytable(self.test_resources + '/sampleAnnotations2.tsv', 'Sample', config = TextTableConfig(impute = True))
+
+        # Variables
+        self.assertEqual(kt.nfields(), 3)
+        self.assertEqual(kt.key_names()[0], "Sample")
+        self.assertEqual(kt.field_names()[2], "qPhen")
+        self.assertEqual(kt.nrows(), 100)
+        kt.schema()
+
+        # Export
+        kt.export('/tmp/testExportKT.tsv')
+
+        # Filter, Same
+        ktcase = kt.filter('Status == "CASE"', True)
+        ktcase2 = kt.filter('Status == "CTRL"', False)
+        self.assertTrue(ktcase.same(ktcase2))
+
+        # Annotate
+        kt4 = kt.annotate('X = Status', 'Sample, Status')
+
+        # Join
+        kt5 = kt.join(kt2, 'left')
+
+        # AggregateByKey
+        kt6 = kt.aggregate_by_key("Status = Status", "Sum = qPhen.sum()")
```

Comment:
do you want to actually evaluate these (count or something), or is that sufficiently handled in the scala tests?